src/app/_services/googleCalendar.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from "@angular/core";
+declare var gapi: any;
+
+interface AppointmentEvent {
+    startDateTime: Date;
+    endDateTime: Date;
+    doctor: String;
+}
+
+@Injectable()
+export class GoogleCalendarService {
+
+	static clientId = '261952745745-e55n2f6jg7mn1pfacvv0iggchdmse2nc.apps.googleusercontent.com';
+	static apiKey = 'AIzaSyCNsq6DWKCERx7iCDn8-gMY90tLtNlhSTA';
+	static scope = 'https://www.googleapis.com/auth/calendar';
+	static discoveryDocURLs = ['https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest'];
+	
+	//public isInitialized: boolean = false;
+	public isInitialized: boolean = true;
```

Comment:
To wygląda jakby miało być 'false' na początku

Reply:
Tak jak napisałem wyżej, przesunąłem inicjalizację do index.html i zapomniałem to usunąć. Jeśli chcesz to to wywalę